lib/server.js

```diff
@@ -92,6 +92,14 @@ Server.prototype.start = function () {
   this._injector.invoke(this._start, this)
 }
 
+/**
+ * Backward-compatibility with karma-intellij bundled with WebStorm
+ * @deprecated since version 0.13, to be removed in 0.14
+ */
+Server.start = function (config) {
```

Comment:
Ah, I wasn't aware of it: `karma-intellij` requires `./lib/server.js` directly, so the fix is working for karma-intellij. Agreed, the fix should be more general. Will update the PR soon.